hack/make.sh

```diff
@@ -50,7 +50,11 @@ function main ()
 	# wait until pouch daemon is ready
 	while true;
 	do
-		if [ -S /var/run/pouchd.sock ];then
+	        count=`ps -ef | grep pouchd | grep -v grep | wc -l`
```

Comment:
please format the shell code, tag or whitespace. Please keep the same.

Reply:
I will change count to COUNT, but I don't know how to format white space. Is there a shell code style doc ?

Reply:
Please use "2 tabs" before "count", not space.